test/wdio/lifecycle-async/util.ts

```diff
@@ -0,0 +1,5 @@
+export function timeout(ms: number): Promise<void> {
```

Comment:
Do we need this?

Reply:
yeah two of the components use it, it's not used in the test file but in `cmp-b` and `cmp-c`

Reply:
ah, ok didn't recognize that. I thought it was a way to "pause" the test. All good then.